### ISSUE TITLE
Generate and mirror map data

### ISSUE THREAD

**Issue by maxammann** [loeli-style/loeli-style#7]
There is an interest in downloading and self-hosting map data: https://www.reddit.com/r/openstreetmap/comments/v9yoed/comment/j8lvfyr/?utm_source=share&utm_medium=web2x&context=3

If we find about 10 independent entities which would contribute mirrors in all geographic areas, then we could build a sustainable network.

Note that we need at least 2 entities which generate map data. The more the better.

**Comment by wipfli** [comment]
How does IPFS work by the way? I think I saw Hidenori (https://www.linkedin.com/in/hidenori-fujimura-7a273711/) experiment with that in Japan.

**Comment by wipfli** [comment]
Or we can have the pmtiles file on Cloudflare R2. There you don't pay for outbound traffic.

**Comment by wipfli** [comment]
Pmtiles and cloudflare is very simple to set up and requires basically no maintenance. If we should ever be unhappy with cloudflare, we can just go to any other solution that supports http range requests...

**Comment by maxammann** [comment]
Sure, one of the mirrors can use cloudflare. But we need at least a few mirrors nonetheless. Having multiple organsiations/people who support the project makes it last.

e.g. if at some point we do not want to continue hosting then there are still other people out there who supply users with the data.

This is more a people-problem than a technical problem.

I'm not 100% certain which exact problem IPFS solves. E.g. is it an alternative to the torrent protocol? But I guess it's definitely more complex than running 10 Cloudflare R2 stores or 10 nginx instances.

**Comment by wipfli** [comment]
You are right, this is a people problem. Yeah let's build a small network of mirrors.

**Comment by wipfli** [comment]
I have a 1 gbps hetzner in Germany we can use...
